### ISSUE TITLE
Tooltips not showing on nav highlighting but follow nav highlighting on mouse hover

### ISSUE THREAD

**Issue by euyogi** [ocornut/imgui#6622]
**Version/Branch of Dear ImGui:**

Version: 1.89.7 (18971)
Branch: docking_

**Back-end/Renderer/Compiler/OS**

Back-ends: imgui_impl_win32cpp + imgui_impl_dx12.cpp

**My Issue/Question:**

Hello, not sure if I did something that made it behave like that or if it's intended but when ImGui::IsItemHovered() uses ImGuiHoveredFlags_ForTooltip, tooltips won't show on nav highlight, but if cursor is hovering the tooltip will show correctly (on the cursor) but if you move the nav highlight the tooltip will follow it even outside the item that the tooltip should be.

**Video**

https://github.com/ocornut/imgui/assets/46427886/f7d76610-6efe-4066-8d26-ae48dffefb18

**Standalone, minimal, complete and verifiable example:** 

You can check it on ImGui Demo Window > Widgets > Tooltips.

Basic, Fancy and Manual tooltips don't show tooltip on  nav highlight, the others do. Those same tooltips if hovered by the cursor and moving the nav highlight, the tooltip will follow it, the others don't behave like that. I just used without the ImGuiHoveredFlags_ForTooltip on my project so it isn't much of a problem.

**Comment by ocornut** [comment]
Ouch. Very embarrassingly when I added `ImGuiHoveredFlags_AllowWhenOverlappedByItem` for 10c7709f30 (June 26) i offset some flags and created a conflict between `ImGuiHoveredFlags_ForTooltip` and `ImGuiHoveredFlags_NoNavOverride`.

Big ouch. Pushed the fix 1109de3. Thank you!

**Comment by ocornut** [comment]
Unrelated but I am confused as to why your RadioButton() items look so angly.

- Have you changed the value for `style.CircleTessellationMaxError` ? Default is 0.30. Higher value means lower quality.
- Or are you using some elaborate/custom overall scaling scheme?

**Comment by euyogi** [comment]
Yeah, I changed style.CircleTessellationMaxError, was messing around heh
